### ISSUE TITLE
"[object Object]" - missing label placeholder being read by screenreader

### ISSUE THREAD

**Issue by jvine** [ProjectMirador/mirador#2801]
This is appearing 2 places that I can find, and it looks like missing text?

1. Landmark label for the Search panel:
`<aside class="MuiPaper-root MuiPaper-elevation1 jss636 jss640 jss630 mirador-companion-window-left " aria-label="[object Object]" style="order: -1; position: relative;">`
This label should be `Search`.

2. Full screen button:
`<button class="MuiButtonBase-root MuiIconButton-root jss211" tabindex="0" type="button" i18n="[object Object]" aria-label="Full screen" title="Full screen">`
I don't know what the `i18n` value does, so I'm not sure what should be here.
